static/templates/user_info_popover_content.hbs

```diff
@@ -43,6 +43,24 @@
         {{else}}
             <li>{{ user_type }}</li>
         {{/if}}
+        {{!-- Display selected custom profile fields in this popover. --}}
+        {{#each display_profile_fields}}
+            <li data-type="{{this.type}}" class="field-section custom_user_field tippy-zulip-tooltip" data-tippy-content="{{this.name}}" data-field-id="{{this.id}}">
+                {{#if this.is_link}}
+                    <a tabindex="0" href="{{this.value}}" target="_blank" rel="noopener noreferrer" class="value">{{this.value}}</a>
+                {{else if this.is_external_account}}
+                    <a tabindex="0" href="{{this.link}}" target="_blank" rel="noopener noreferrer" class="value">
+                        {{this.value}}
+                    </a>
+                {{else}}
+                    {{#if this.rendered_value}}
+                    <span class="value rendered_markdown">{{rendered_markdown this.rendered_value}}</span>
+                    {{else}}
+                    <span class="value">{{this.value}}</span>
+                    {{/if}}
+                {{/if}}
+            </li>
+        {{/each}}
```

Comment:
This HTML block very much duplicates a block in `user_profile_modal.hbs` ; this is probably not desired but it may require some care to refactor, so I'm OK merging this as written. 

@sahil839 FYI in case you have ideas for how to deduplicate this nicely.